build/yamls/antrea-windows.yml

```diff
@@ -255,11 +255,16 @@ spec:
           path: /k/antrea
           type: DirectoryOrCreate
         name: host-antrea-home
-      - hostPath:
-          path: /
-        name: host
       - hostPath:
           path: \\.\pipe\rancher_wins
         name: wins
+      - hostPath:
+          path: /var/log/antrea/
+          type: DirectoryOrCreate
+        name: var-log-antrea
```

Comment:
so this is just to ensure the directory is created, we don't actually use the mount?

Reply:
For running Antrea Agent on Windows daemonset case, we don't access the log path inside the management containers. But we should ensure it exists on the host, and pass it to wins to run agent process on the host

Reply:
Yes.